### PR TITLE
feat(proxy-cache): add ignore_uri_case to configuring cache-key uri to be handled as lowercase

### PR DESCRIPTION
**Pull request by tobiasehlert**
### Description

This PR adds documentation of `ignore_uri_case` parameter in the cache-proxy plugin added with PR https://github.com/Kong/kong/pull/10453.

### Testing instructions

Netlify link: https://deploy-preview-5270--kongdocs.netlify.app

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`release/gateway-3.3`)

### CODE REVIEW

**Review by cloudjumpercat — COMMENTED** [review]
Thanks for making these changes! I made a small suggestion for wording if you want to see if that's accurate. I also added a label "blocked by kong", so we'll wait until your kong repo PR is merged before we merge this one.

**Comment by tobiasehlert** [comment]
@cloudjumpercat now https://github.com/Kong/kong/pull/10453 got merged :)

Something to change on this PR like switching from main to some other branch?
This is probably coming with Kong release 3.3.0

**Comment by cloudjumpercat** [comment]
@tobiasehlert Thanks for letting me know!

@samugi Do you know if https://github.com/Kong/kong/pull/10453 will be going out with Gateway 3.3? I didn't see any labels on the PR, so I wasn't sure.

File: app/_hub/kong-inc/proxy-cache-advanced/_index.md
Comment (samugi) [inline app/_hub/kong-inc/proxy-cache-advanced/_index.md:95]: Please revert this, proxy-cache-advanced is a different plugin and will be treated separately

File: app/_hub/kong-inc/proxy-cache/_index.md
Comment (samugi) [inline app/_hub/kong-inc/proxy-cache/_index.md:96]: Let's make this description more user-friendly
```suggestion
        If set to `true`, requests will be cached while ignoring case sensitivity in the URI.
```

**Review by samugi — CHANGES_REQUESTED** [review]

**Comment by samugi** [comment]
> @tobiasehlert Thanks for letting me know!
> 
> @samugi Do you know if [Kong/kong#10453](https://github.com/Kong/kong/pull/10453) will be going out with Gateway 3.3? I didn't see any labels on the PR, so I wasn't sure.

yep, 3.3

File: app/_hub/kong-inc/proxy-cache-advanced/_index.md
Comment (tobiasehlert) [reply app/_hub/kong-inc/proxy-cache-advanced/_index.md:95]: Ah alright, didn't know there was some other process for that. I assumed that this was supposed to be added there as well :-)

Comment (tobiasehlert) [reply app/_hub/kong-inc/proxy-cache-advanced/_index.md]: resolved with d03e05e5e33933a92c49c069f96971934ededc78

**Review by samugi — APPROVED** [review]

**Comment by cloudjumpercat** [comment]
@tobiasehlert Could you change the base branch of this PR to `release/gateway-3.3`? Then we can get it merged and it'll be published with the Gateway 3.3 release.

**Comment by tobiasehlert** [comment]
> @tobiasehlert Could you change the base branch of this PR to `release/gateway-3.3`? Then we can get it merged and it'll be published with the Gateway 3.3 release.

Done :)

**Comment by cloudjumpercat** [comment]
@tobiasehlert Thanks for doing that! Sorry for not catching this sooner, but it looks like the only thing remaining to do before this can be merged is to add `ignore_uri_case` to the dictionary file for Vale. Looks like that's what's causing the Vale check failures. 

To do that, all you'd have to do is add `ignore_uri_case` to this file: https://github.com/Kong/docs.konghq.com/blob/main/.github/styles/kongplugins/plugin-ignore.txt

**Comment by tobiasehlert** [comment]
@cloudjumpercat done! Now all checks are green 🥳

**Review by cloudjumpercat — APPROVED** [review]
LGTM! Thanks for making these updates!

**Pull request merged**
